SYCL/ESIMD/api/functional/operators/common.hpp

```diff
@@ -0,0 +1,51 @@
+//===-- common.hpp - Define common code for simd operators tests ----------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+///
+/// \file
+/// This file provides common things for simd operators tests.
+///
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "../common.hpp"
+
+namespace esimd_test::api::functional::operators {
+
+// Dummy kernel for submitting some code into device side.
```

Comment:
ditto

Reply:
Unused kernels has been removed in this commit: e8b81483455f0a82e3d719185b184e3c2dd1715c